docs/assets/css/styles.scss

```
@@ -0,0 +1,32 @@
+---
+# this ensures Jekyll reads the file to be transformed into CSS later
+# only Main files contain this front matter, not partials.
+---
+
+.code-header {
+  display: flex;
+  justify-content: flex-end;
```

I don't know how flex works because I haven't done frontend for years, but this removes the extra spacing:
```suggestion
  float: right;
```

---

Solved!